app/src/androidTest/java/com/example/househomey/SelectDeleteTest.java

```diff
@@ -0,0 +1,2 @@
+package com.example.househomey;public class SelectDeleteTest {
+}
```

Comment:
looks like you didn't actually commit the test? Just empty class somehow

Reply:
OMG, yeah I realized I pushed the wrong commit. Yeah I'll commit the test and push lol